src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -172,7 +172,16 @@ export default class MiniScatterChart<
         };
     }
 
+    getReversedData() {
```

Comment:
you want to use a computed getter here to avoid redundant recomputation on every render cycle.  see ```@computed``` attribute elsewhere in codebase for example.

Reply:
It seems odd that we always want to consume "reversed" data and that would be a fix for the issue.  shouldn't this be conditional based on group order?